recipes/libselinux/all/conanfile.py

```diff
@@ -87,7 +87,9 @@ def generate(self):
         sepol_lib_folder = os.path.join(self._sepol_source_folder, "src")
         tc.extra_ldflags.append(f"-L{sepol_lib_folder}")
         tc.make_args.append("USE_PCRE2=y")
-        tc.generate()
+        env = tc.environment()
+        env.append_path("PKG_CONFIG_LIBDIR", self.generators_folder)
+        tc.generate(env=env)
```

Comment:
Should we apply this only for cross-compilations?

Reply:
I don't see why not. I've updated this.